gamestonk_terminal/economy/wsj_view.py

```diff
@@ -0,0 +1,48 @@
+"""WSJ view module"""
+__docformat__ = "numpy"
+
+from tabulate import tabulate
+
+from gamestonk_terminal.wsj_functions import (
+    market_overview,
+    us_bonds,
+    us_indices,
+    global_bonds,
+    global_currencies,
+    top_commodities,
+)
+
+
+def display_wsj(fn: str):
```

Comment:
Can you change this function to handle each command separately, and add the usual helper stuff. This allows to keep the helper functionality, and have the code more "explicit". Otherwise the way I see this is:
```
market            --\                                 / market
us_bonds        --->     display_wsj  ---- us_bonds
global_bonds  -- /                               \ global_bonds
```

which looks like we're adding complexity. I would only have a display_wsj like that if we got all the data from 1 single function as well.

Reply:
well what helper stuff is there?  All this display_wsj function does is print the tabulate function.  I would just do it in the controller, but that goes against the structure.  So just make 6 individual functions that just load different data?

Reply:
The usual helper stuff is this 
```
parser = argparse.ArgumentParser(
        prog="darkpos",
        add_help=False,
        description="Shows Net Short Vol. vs Position. [Source: Stockgrid]",
    )
```

Yea, I'd say to keep it on wsj_view but have 6 functions for it.